packages/ckeditor5-ui/src/index.ts

```diff
@@ -17,7 +20,7 @@ export { default as AccessibilityHelp } from './editorui/accessibilityhelp/acces
 
 export { default as BodyCollection } from './editorui/bodycollection.js';
 
-export { default as Button, type ButtonExecuteEvent } from './button/button.js';
+export type { default as Button, ButtonExecuteEvent } from './button/button.js';
```

Comment:
Why? Isn't it breaking change?

Reply:
No, because `Button` is an interface and `ButtonExecuteEvent` is a type.